install/example-cws-configuration.properties

```diff
@@ -116,6 +116,9 @@ cws_notification_emails=[AdminEmailAddresses(comma separated)]
 # This allows CWS to use a user provided Elasticsearch service.
 # If you wish to use an unsecured Elasticsearch host, you may do so
 # by specifying elasticsearch_use_auth=n below.
+# Also specify if your provided Elasticsearch is encrypted(https) or encrypted(http)
+# by setting elasticsearch_protocol to https or http.
+elasticsearch_protocol=[YourElasticsearchProtocol]
```

Comment:
Add the choices.  so they can see the possible choices

Reply:
'http://' or 'https://'

Reply:
http or https works too

Reply:
Make the examples and documentation with "HTTP" or "HTTPS" only

Reply:
Documentation and Interactive mode prompts display uppercase HTTP/HTTPS